sql/core/src/test/scala/org/apache/spark/sql/sources/InsertSuite.scala

```diff
@@ -1027,13 +1031,13 @@ class InsertSuite extends DataSourceTest with SharedSparkSession {
     }
     withTable("t") {
       sql("create table t(i int, s bigint default 42, x bigint) using parquet")
-      sql("insert into t values(1)")
```

Comment:
which Spark version added this test?

Reply:
Added by https://github.com/apache/spark/pull/41262 which haven't been released yet.